### ISSUE TITLE
uncaughtException in Node: OpenError

### ISSUE THREAD

**Issue by probonopd** [digidem/mapeo-desktop#185]
```
me@host:~$ Downloads/Installar_Mapeo_v3.1.3_linux.AppImage 
20024
ws prepped
[STARTUP] Listening for uncaught exceptions
uncaughtException in Node: { Error: listen EADDRINUSE :::17509
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at Server.setupListenHandle [as _listen2] (net.js:1351:14)
    at listenInCluster (net.js:1392:12)
    at Server.listen (net.js:1476:7)
    at websocketServer (/tmp/.mount_InstaltkYeQa/resources/app.asar/src/lib/log.js:85:10)
    at setupNodeLogger (/tmp/.mount_InstaltkYeQa/resources/app.asar/src/lib/log.js:49:3)
    at Object.module.exports.Node (/tmp/.mount_InstaltkYeQa/resources/app.asar/src/lib/log.js:16:18)
    at Object.<anonymous> (/tmp/.mount_InstaltkYeQa/resources/app.asar/src/lib/locale.js:6:33)
    at Object.<anonymous> (/tmp/.mount_InstaltkYeQa/resources/app.asar/src/lib/locale.js:35:3)
  code: 'EADDRINUSE',
  errno: 'EADDRINUSE',
  syscall: 'listen',
  address: '::',
  port: 17509 }
uncaughtException in Node: OpenError: IO error: lock /home/me/.config/Mapeo/data/index/LOCK: Resource temporarily unavailable
    at /tmp/.mount_InstaltkYeQa/resources/app.asar/node_modules/levelup/lib/levelup.js:117:34
    at /tmp/.mount_InstaltkYeQa/resources/app.asar/node_modules/abstract-leveldown/abstract-leveldown.js:39:16
```

**Comment by karissa** [comment]
Do you have another instance of Mapeo currently running?

**Comment by karissa** [comment]
and does the folder `home/me/.config/Mapeo/data` exist on your machine and accessible? perhaps you need to run as root

**Comment by probonopd** [comment]
No, I am running this on a new Ubuntu 14.04 instance on Travis CI. No pre-existing folder or instance. But network connections are disallowed by [Firejail](https://firejail.wordpress.com/).

**Comment by benhur07b** [comment]
Hi! Had the same issue before (Mapeo 3.1.3 on Xubuntu 18.04). What I did was make sure that there were no other instances of mapeo-desktop running. 

To check I ran: **sudo netstat -apv | grep mapeo-desktop**
I then killed all instances with: **sudo killall mapeo-desktop**

Afterwards, I went to the directory with the AppImage and ran it: **sudo ./Installar_Mapeo_v3.1.3_linux.AppImage**

This also worked for v3.1.4. I'm still testing the .deb file if it's easier to use.

I hope this helps.

Cheers!
